tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2019",
-    "lib": ["es2020"],
+    "lib": ["es2022"],
```

Comment:
`Object.hasOwn` works in our version of Node, so I believe this is a safe change to make.